views/_snippets.html

```diff
@@ -195,3 +278,500 @@ <h3 class="package-title-{{package.datahub.findability}}">
     });
   </script>
 {%- endmacro %}
+
+{% macro defaultResults() -%}
+<div class="package-summary-list">
+
+    <!-- search result -->
+    <div class="row">
+        <div class="col-sm-12 pad-xs-left-zero pad-xs-right-zero ">
+            <div class="square">
+              <div class="content">
+                <div class="row">
+                  <div class="col-sm-8 pad-xs-left-zero pad-xs-right-zero">
+                    <a href="/core/world-cities"><h3>Major cities of the world</h3></a>
+                  </div>
+                  <div class="col-sm-4 pad-xs-left-zero pad-xs-right-zero">
+                      <div class="meta">
+                      </div>
+                    <span class="pull-right">
+                          updated 3 weeks ago
```

Comment:
we should not hardcode this info. I know that this is shown by default without querying metastore but probably we should say "updated regularly" instead of 3 weeks etc.